### PR TITLE
feat: new list items for doc type

### PR DESCRIPTION
**Pull request by harshil1793**
## Summary

- *This work is behind a feature toggle (flipper): YES/NO* - No
- This PR adds more options to the document types
- *(If bug, how to reproduce)* - NA
- *(What is the solution, why is this the solution?)* - NA
- *(Which team do you work for, does your team own the maintenance of this component?)* - Claims-status
- *(If introducing a flipper, what is the success criteria being targeted?)*

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/87899

## Testing done

- [ ] *New code is covered by unit tests* - NA
- *Describe what the old behavior was prior to the change* NA
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing* NA
- *If this work is behind a flipper:* NA
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).* NA
  - *What is the testing plan for rolling out the feature?* NA

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)* - NA

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: app/models/evss_claim_document.rb [hunk 1]
```diff
@@ -24,33 +24,39 @@ class EVSSClaimDocument < Common::Base
 
   # rubocop:disable Layout/LineLength
   DOCUMENT_TYPES = {
-    '1489' => 'Hypertension Rapid Ready For Decision Summary PDF',
```
Comment (jerekshoe) [inline app/models/evss_claim_document.rb:27]: These should be ordered by the L value, not the description string

File: app/models/evss_claim_document.rb [hunk 2]
```diff
@@ -24,33 +24,39 @@ class EVSSClaimDocument < Common::Base
...
+    'L1489' => 'Automated Review Summary Document',
+    'L014' => 'Birth Certificate',
     'L015' => 'Buddy/Lay Statement',
     'L018' => 'Civilian Police Reports',
-    'L023' => 'Other Correspondence',
     'L029' => 'Copy of a DD214',
-    'L034' => 'Military Personnel Record',
+    'L418' => 'Court papers / documents'
```
Comment (jerekshoe) [inline app/models/evss_claim_document.rb:32]: Make sure you aren't missing any commas

File: app/models/evss_claim_document.rb [hunk 3]
```diff
@@ -24,33 +24,39 @@ class EVSSClaimDocument < Common::Base
...
+    'L033' => 'Death Certificate',
+    'L702' => 'Disability Benefits Questionnaire (DBQ)',
+    'L037' => 'Divorce Decree',
+    '' => 'Financial Information',
```
Comment (jerekshoe) [inline app/models/evss_claim_document.rb:36]: I think you can remove this one

File: app/models/lighthouse_document.rb [hunk 4]
```diff
@@ -28,33 +28,39 @@ class LighthouseDocument < Common::Base
 
   # rubocop:disable Layout/LineLength
   DOCUMENT_TYPES = {
-    '1489' => 'Hypertension Rapid Ready For Decision Summary PDF',
```
Comment (jerekshoe) [inline app/models/lighthouse_document.rb:31]: Same thing here, order these by L value not by the description

File: app/models/evss_claim_document.rb (hunk 1 above)
Comment (harshil1793) [reply app/models/evss_claim_document.rb:27]: Thanks for the feedback! However, the acceptance criteria specify that the dropdown list should be alphabetized. If this refers to visual ordering, sorting by the L value won't achieve the same result. Could you please confirm?

Comment (jerekshoe) [reply app/models/evss_claim_document.rb:27]: This list doesn’t affect the frontend so the way it is ordered doesn’t matter. The frontend has it’s own copy of this list and needs to be updated accordingly

Comment (harshil1793) [reply app/models/evss_claim_document.rb:27]: That makes sense, let me push the change.

File: app/models/evss_claim_document.rb [hunk 5]
```diff
@@ -24,33 +24,38 @@ class EVSSClaimDocument < Common::Base
...
-    'L228' => 'VA Form 21-0781 - Statement in Support of Claim for PTSD',
-    'L229' => 'VA Form 21-0781a - Statement in Support of Claim for PTSD Secondary to Personal Assault',
-    'L450' => 'STR - Dental - Photocopy',
-    'L451' => 'STR - Medical - Photocopy',
-    'L478' => 'Medical Treatment Records - Furnished by SSA',
-    'L702' => 'Disability Benefits Questionnaire (DBQ)',
-    'L703' => 'Goldmann Perimetry Chart/Field Of Vision Chart',
-    'L827' => 'VA Form 21-4142a - General Release for Medical Provider Information',
-    'L1489' => 'Automated Review Summary Document'
+  '1489' => 'Hypertension Rapid Ready For Decision Summary PDF',
```
Comment (jerekshoe) [inline app/models/evss_claim_document.rb:27]: Make sure you are indenting these

File: app/models/lighthouse_document.rb [hunk 6]
```diff
@@ -48,13 +52,14 @@ class LighthouseDocument < Common::Base
     'L222' => 'VA Form 21-0779 - Request for Nursing Home Information in Connection with Claim for Aid & Attendance',
     'L228' => 'VA Form 21-0781 - Statement in Support of Claim for PTSD',
     'L229' => 'VA Form 21-0781a - Statement in Support of Claim for PTSD Secondary to Personal Assault',
+    'L418' => 'Court papers / documents',
     'L450' => 'STR - Dental - Photocopy',
-    'L451' => 'STR - Medical - Photocopy',
+    'L451' => 'STR - Medical - Photocopy'
```
Comment (jerekshoe) [inline app/models/lighthouse_document.rb:57]: You are missing a comma here

File: app/models/lighthouse_document.rb [hunk 7]
```diff
@@ -48,13 +52,14 @@ class LighthouseDocument < Common::Base
...
     'L478' => 'Medical Treatment Records - Furnished by SSA',
     'L702' => 'Disability Benefits Questionnaire (DBQ)',
     'L703' => 'Goldmann Perimetry Chart/Field Of Vision Chart',
     'L827' => 'VA Form 21-4142a - General Release for Medical Provider Information',
-    'L1489' => 'Automated Review Summary Document'
+    'L1489' => 'Automated Review Summary Document',
```
Comment (jerekshoe) [inline app/models/lighthouse_document.rb:62]: You don't need a comma here

**Review by samcoforma — APPROVED** [review]

**Review by cilestin — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]

**Pull request merged**